src/cpp/shared/game/game.cpp

```diff
@@ -301,7 +301,7 @@ void game::ProcessOnMapHeroes() {
         }
         randomHero = &this->heroes[mapExtraHero->heroID];
 
-        if (!isJail && mapExtraHero->relatedToName[19]) {
+        if (!isJail && mapExtraHero->name[19]) {
```

Comment:
This is a memory error; name has length 13 now.

Reply:
It gets fixed later.